### DISCUSSION TITLE
buddhism/heart_sutra_details/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/buddhism#51] [General]
# buddhism/heart_sutra_details/

https://shanenull.com/buddhism/heart_sutra_details/

**Comment by shane0** [comment]
[diamond sutra](https://shanenull.com/buddhism/diamond_sutra/)

**Comment by shane0** [comment]
https://shanenull.com/buddhism/skhandas/

**Comment by shane0** [comment]
https://shanenull.com/buddhism/sense/
